src/Symfony/Component/PropertyInfo/Tests/Extractors/ReflectionExtractorTest.php

```diff
@@ -69,7 +69,7 @@ public function typesProvider()
             array('b', array(new Type(Type::BUILTIN_TYPE_OBJECT, true, 'Symfony\Component\PropertyInfo\Tests\Fixtures\ParentDummy'))),
             array('c', array(new Type(Type::BUILTIN_TYPE_BOOL))),
             array('d', array(new Type(Type::BUILTIN_TYPE_BOOL))),
-            array('e', null),
+            array('e', array(new Type(Type::BUILTIN_TYPE_ARRAY, false, null, true, new Type(Type::BUILTIN_TYPE_INT)))),
```

Comment:
Sure, but why [this line](https://github.com/symfony/symfony/pull/20154/files#diff-a015ff68f6929a6d41c7f354ea02e8deL147) and [this one](https://github.com/symfony/symfony/pull/20154/files#diff-a015ff68f6929a6d41c7f354ea02e8deL176) then?